core/src/broadcast_stage.rs

```diff
@@ -416,14 +418,10 @@ pub fn broadcast_shreds(
             let cluster_nodes =
```

Comment:
`broadcast messages from the leader to layer 1 nodes`

I believe this comment is no longer accurate. Should be leader to root

Reply:
done.